src/Evolution/Systems/GeneralizedHarmonic/GaugeSourceFunctions/DampedHarmonic.cpp

```diff
@@ -742,6 +753,31 @@ void spacetime_deriv_damped_harmonic_h(
       d4_gauge_h->get(a, b) = dT1.get(a, b) + dT2.get(a, b) + dT3.get(a, b);
     }
   }
+
+  // In case we want SpEC-style roll-on of gauge
+  if (UNLIKELY(use_spec_style_rollon)) {
+    // Calculate T2 + T3
+    typename db::item_type<Tags::GaugeH<SpatialDim, Frame>> _T2_plus_T3{
+        get_size(get(lapse))};
+    const auto _zero_h_init = make_with_value<
+        typename db::item_type<Tags::GaugeH<SpatialDim, Frame>>>(lapse, 0.);
```

Comment:
remove the leading underscores from variable names

Reply:
removed